README.md

```diff
@@ -1,49 +1,71 @@
-	- [**CaptureHDRLoop**](https://github.com/zivid/cpp-extra-samples/tree/master/Zivid/CaptureHDRLoop/CaptureHDRLoop.cpp) - This example shows how to acquire HDR images from the Zivid camera in a loop (while actively changing some HDR settings).
 	- [**CaptureSavePLY**](https://github.com/zivid/cpp-extra-samples/tree/master/Zivid/CaptureSavePLY/CaptureSavePLY.cpp) - This example shows how to capture a Zivid point cloud and save it to a .PLY file format.
-	- [**ConnectToSerialNumberCamera**](https://github.com/zivid/cpp-extra-samples/tree/master/Zivid/ConnectToSerialNumberCamera/ConnectToSerialNumberCamera.cpp) - This example shows how to connect to a specific Zivid camera based on its serial number.
-	- [**GetCameraIntrinsics**](https://github.com/zivid/cpp-extra-samples/tree/master/Zivid/GetCameraIntrinsics/GetCameraIntrinsics.cpp) - This example shows how to read the intrinsic calibration parameters of the Zivid camera (OpenCV model).
-	- [**ReadZDF**](https://github.com/zivid/cpp-extra-samples/tree/master/Zivid/ReadZDF/ReadZDF.cpp) - This example shows how to import and display a Zivid point cloud from a .ZDF file.
+	- [**ConnectToSerialNumberCamera**](https://github.com/zivid/cpp-extra-samples/tree/master/Zivid/ConnectToSerialNumberCamera/ConnectToSerialNumberCamera.cpp) - This example shows how to connect to a specific Zivid camera based on its serial number.	
 	- [**ZDF2PLY**](https://github.com/zivid/cpp-extra-samples/tree/master/Zivid/ZDF2PLY/ZDF2PLY.cpp) - This example shows how to convert a Zivid point cloud from a .ZDF file format to a .PLY file format.
 
 - [**ZividPCL**](https://github.com/zivid/cpp-extra-samples/tree/master/ZividPCL)
 	- [**ZDF2PCD**](https://github.com/zivid/cpp-extra-samples/blob/master/ZividPCL/ZDF2PCD/ZDF2PCD.cpp) - This example shows how to convert a Zivid point cloud from a .ZDF file format to a .PCD file format.
-	- [**ReadPCLVis3D**](https://github.com/zivid/cpp-extra-samples/blob/master/ZividPCL/ReadPCLVis3D/ReadPCLVis3D.cpp) - This example shows how to read a PCL point cloud and visualize it.
-	- [**CaptureWritePCLVis3D**](https://github.com/zivid/cpp-extra-samples/blob/master/ZividPCL/CaptureWritePCLVis3D/CaptureWritePCLVis3D.cpp) - This example shows how capture a Zivid point cloud, save it to a .PCD file format, and visualize it.
 	- [**CaptureFromFileWritePCLVis3D**](https://github.com/zivid/cpp-extra-samples/blob/master/ZividPCL/CaptureFromFileWritePCLVis3D/CaptureFromFileWritePCLVis3D.cpp) - This example shows how capture a Zivid point cloud from an emulated Zivid camera, save it to a .PCD file format, and visualize it.
 	- **Dependencies:**
 		- [PCL](http://pointclouds.org/) version 1.9.1
 
-- [**Downsample**](https://github.com/zivid/cpp-extra-samples/tree/master/Downsample)  - This example shows how to import a Zivid point cloud from a .ZDF file and downsample it.
-	- **Dependencies:**
-		- [Eigen](http://eigen.tuxfamily.org/) version 3.3.7 or newer
+## Samples renamed
```

Comment:
...and this.

Reply:
It's for your eyes only :) I guess the fact that these are renamed will appear in a git commit